README.md

```diff
@@ -66,12 +70,12 @@ A great song like "Livin' on a Prayer" would have a rating of 10 and appear gree
 Without getting into specifics of how you would write the code itself, pick one and defend your answer:
 
 This code would make most sense as a...
-- ...model method.
+X ...model method.
 - ...controller method.
 - ...helper method.
 
 ```text
-Your answer...
+Because `avg_rating` is an attribute stored in the database, the songs color can be determined without ever having to know anything the controller knows.  Separation of concerns.
 ```
```

Comment:
this would actually be a helper, as those "help" render html based on conditions